pkg/reconciler/v1alpha1/configuration/config/store.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	"github.com/knative/serving/pkg/reconciler/config"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	RevisionGcConfigName = "config-revision-gc"
```

Comment:
Can you add tests for this config?

Also, we symlink the actual configmap into `testdata/` and have tests that the configmaps can be loaded without errors.